includes/class-press-sync.php

```diff
@@ -13,6 +13,7 @@
  * @since 0.1.0
  */
 class Press_Sync {
+	const PAGE_SIZE = 1;
```

Comment:
page_size is set in multiple files. Are they the same var? or it's different for each class?